src/Configs/ConfigFactory.php

```diff
@@ -15,17 +15,12 @@ class ConfigFactory
         '__files' => System::class,
         'aliases' => System::class,
         'packages' => System::class,
-        'dotenv' => DotEnv::class,
+        'envs' => Envs::class,
```

Comment:
Let's keep all the variants, cause we depend on it greatly.
At least until we'll change it all around all our code.
```suggestion
        'envs' => Envs::class,
        'dotenv' => Envs::class,
        'defines' => Constants::class,
```

Reply:
We can make it for 2 iterations to make the transition easly:
1. Keep 2 variants and mark old as deprecated
2. Make new version and remove old variants. You will have a lot of time to migrate to new version
 WDYT?

Reply:
Sounds OK.